ote/src/cljs/ote/views/admin/detected_changes.cljs

```diff
@@ -97,19 +105,41 @@
        [:span "Pakota yhden lataamattoman pakettin lataus ulkoisesta osoitteesta"]]]]
 
     [:h2 "Muutostunnistuksen käynnistys"]
+    [:div
+     [:div (stylefy/use-style style-admin/detection-button-container)
+      [:div (stylefy/use-style style-admin/detection-info-text)
+       [:span "Pakota muutostunnistus kaikkille palveluille. Tämä vie noin 3 minuuttia."]]
+      [:div {:style {:flex 2}}
+       [:a (merge (stylefy/use-style button-styles/primary-button)
+                  {:id       "force-detect-transit-changes"
+                   :href     "#"
+                   :on-click #(do
+                                (.preventDefault %)
+                                (e! (admin-transit-changes/->ForceDetectTransitChanges)))
+                   :icon     (ic/content-filter-list)})
+        [:span "Käynnistä muutostunnitus"]]]]
 
-    [:div (stylefy/use-style style-admin/detection-button-container)
-     [:div (stylefy/use-style style-admin/detection-info-text)
-      [:span "Pakota muutostunnistus kaikkille palveluille. Tämä vie noin 3 minuuttia."]]
-     [:div {:style {:flex 2}}
-      [:a (merge (stylefy/use-style button-styles/primary-button)
-                 {:id "force-detect-transit-changes"
-                  :href "#"
-                  :on-click #(do
-                               (.preventDefault %)
-                               (e! (admin-transit-changes/->ForceDetectTransitChanges)))
-                  :icon (ic/content-filter-list)})
-       [:span "Käynnistä muutostunnitus"]]]]
+     [:div (stylefy/use-style style-admin/detection-button-container)
+      [:div {:style {:flex 2}} "Käynnistä muutostunnistus vain yhdelle palvelulle. Anna kenttään palvelun id.
+      Löydät palvelun id:n omat palvelutiedot sivun kautta tai muutostunnistuksen visualisointisivun url:stä."]
+      [:div {:style {:flex 2}}
+       [ui/text-field
+        {:id                  "jaakko-mies"
+         :name                "jes"
+         :floating-label-text "Palvelun id"
+         :value               (get-in app-state [:admin :transit-changes :single-detection-service-id])
+         ;:type    :string
+         :on-change           #(do
+                                 (.preventDefault %)
+                                 (e! (admin-transit-changes/->SetSingleDetectionServiceId %2)))}]
+       [:a (merge (stylefy/use-style button-styles/primary-button)
```

Comment:
Is there a specific reason for using `[:a` instead of a `button`? Buttons should be used for actions, links for navigation.
Could `save` in `ote/ui/buttons.cljs` be used, if not then define an overload button there?